internal/parser/grammar.md

```diff
@@ -88,9 +88,16 @@ The parser directly implements the following grammar.
     FunctionBodyExpression         = Block | Expression .
     Block                          = "{" StatementList "}" .
     ExpressionList                 = [ Expression { "," Expression } ] .
+    ObjectBody                     = identifier ObjectBodySuffix .
```

Comment:
The `.` indicates the end of an EBNF expression so it is incorrect to use one on this line since it is followed by an alternate operator `|`. Also, the empty expression needs to be available too so you probably need:

```
ObjectLiteral = "{" [ ObjectBody ] "}"
```

I don't generally like putting so many things on the same line so you may want to do something like this:

```
ObjectLiteral = "{" ObjectLiteralBody "}"
ObjectLiteralBody = [ ObjectBody ]
```

The only thing `ObjectLiteralBody` would do is check if there is more to parse.